bool continuous
bool reset
---
string handpose
